util/toolkit/src/commands/dust_balance.rs

```diff
@@ -33,7 +33,7 @@ pub struct GenerationInfoPair {
 pub struct DustBalanceJson {
 	generation_infos: Vec<GenerationInfoPair>,
 	source: HashMap<String, u128>,
-	total: u128,
+	pub total: u128,
```

Comment:
not sure about this

Reply:
I think that's fine, no need to hide fields here - may be better to make them all `pub` for future proofing